### PR TITLE
Switch to cli for bundling

### PR DESCRIPTION
**Pull request by daun**
**Description**

- Switch to `@swup/cli` for bundling
- Correctly separate dependencies from dev dependencies
- Includes version bump and changelog update

**Checks**

- [x] The PR is submitted to the `master` branch
- [x] The code was linted before pushing (`npm run lint`)
- [ ] ~~The documentation was updated as required~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 <!-- ## [Unreleased] -->
```
Comment (hirasso) [inline CHANGELOG.md]: Should we remove this everywhere?

Comment (daun) [reply CHANGELOG.md]: Makes sense, yeah.

Comment (daun) [reply CHANGELOG.md]: Removed.

**Comment by daun** [comment]
@hirasso Ready for approval :)

**Review by hirasso — APPROVED** [review]

**Pull request merged**